admin/src/scenes/volontaires/list.js

```diff
@@ -492,9 +493,20 @@ const Hit = ({ hit, onClick, selected }) => {
   );
 };
 
-const BadgePhase = ({ text, value }) => (
-  <Badge minify text={text} tooltipText={translate(value)} minTooltipText={`${text}: ${translate(value)}`} color={YOUNG_STATUS_COLORS[value]} />
-);
+const BadgePhase = ({ text, value, id, href }) => {
+  const history = useHistory();
+
+  return (
+    <Badge
+      onClick={() => history.push(`/volontaire/${id}/${href}`)}
```

Comment:
et ducoup ca donnerait :
```suggestion
const BadgePhase = ({ text, value, redirect }) => {
  const history = useHistory();

  return (
    <Badge
      onClick={() => history.push(redirect)}
```